### ISSUE TITLE
Update `architecture` diagram 

### ISSUE THREAD

**Issue by alrra** [webhintio/hint#1406]
Ref: https://github.com/webhintio/hint/blob/da7cb4903ffd9b36c644d87046598604b2555669/packages/hint/docs/contributor-guide/getting-started/images/architecture.svg

Still contains references to `sonarwhal`.

**Comment by molant** [comment]
We should make this SVG a bit bettter. When you open it there's no text to edit, seems like everything are paths, etc.

**Comment by molant** [comment]
I'm updating a bit the whole page as @CKGrafico had a few suggestions.

This is the new diagram I'm working on. Feedback from anyone is really appreciated:

![image](https://user-images.githubusercontent.com/606594/47393534-9d950500-d6d4-11e8-8d2a-6f430d77ae53.png)

I'm trying to make it more obvious how the events come and go and when each piece gets in play. This should be more aligned with the explanation we have in the page later on.

I tried to keep it as simple as possible, ie. I'm only using a formatter, I've removed all the resource loading process, etc.

Thoughts?
